tests/rptest/tests/license_enforcement_test.py

```diff
@@ -46,8 +46,10 @@ def setUp(self):
     )
     def test_license_enforcement(self, clean_node_before_recovery,
                                  clean_node_after_recovery):
-        installer = self.redpanda._installer
+        self.redpanda.set_environment(
+            {'__REDPANDA_DISABLE_BUILTIN_TRIAL_LICENSE': True})
```

Comment:
I didn't add it in the constructor because I only wanted it to apply to the first two tests but not `test_enterprise_cluster_bootstrap`. `start` and `restart` don't currently take extra environment variables; only extra configs (at the moment).

I guess I could factor out the first two tests into a separate test class, and then I could pass it into the constructor.